vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -154,3 +154,320 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
         [![CustomNavigation]][CustomNavigation]
 
     [CustomNavigation]: ../../assets/user_guides/navigation/custom_navigation.png
+
+
+## Changing Selectors
+
+If you want to have a different selector for dashboard navigation you have to provide the `selector` argument of the [`Navigation`][vizro.models.Navigation] with a different selector.
+Currently available selectors are [`NavBar`][vizro.models.NavBar] and [`Accordion`][vizro.models.Accordion]. If `selector` is not defined it defaults to `Accordion`.
+
+If you want to add icons to your dashboard, you have to provide [`NavBar`][vizro.models.NavBar] model
+
+   ```
+    Dashboard(
+        pages=[page_1, page_2],
+        navigation=Navigation(
+            pages=["My first page", "My second page"]
+            selector=NavBar()
+        )
+    )
+   ```
```

Comment:
```suggestion
If you want to have a different selector for dashboard navigation you have to provide the `selector` argument of the [`Navigation`][vizro.models.Navigation] with a different selector.

   ```
    Navigation(
          pages=["My first page", "My second page"]
          selector=NavBar()
        )
   ```

Currently available selectors are [`NavBar`][vizro.models.NavBar] and [`Accordion`][vizro.models.Accordion]. If `selector` is not defined it defaults to `Accordion`.
```

Might make it a bit more concise, please check the code formatting though haha